docs/content/posts/resources.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Demo Class
+date: 2021-04-11
+lastmod: 2021-04-11
+author: Thomas Vuillaume et al
+cover: /school2021/img/analysis.png
+images:
+  - /school2021/img/analysis.png
+categories:
+  - Other
+tags:
+  - README
+---
+
+Additionnal resources
+
+
+# Additionnal resources
+
+You will find here additionnal resources that have been shared or discussed during the school.
+
+
+## Text editors
+
+Comment in slack by @maxnoe:
+
+First, there are three main different types of text editors:
+- Graphical Text Editors open a new graphical window
+- Terminal Text Editors run inside the terminal
+- Integrated Development Environments (IDEs)
+
+Many people "more advanced" prefer the terminal based editors, because they are very accustomed to working in the terminal and they never need to leave the terminal when using a terminal based editor.
+However, most terminal based editors are not easy to use for beginners and if you are still struggling with getting to know the programming languages that is an additional hurdle you can avoid by using one of the modern graphical editors.
+IDEs are usually Graphical Text editors with many, many features concerning one or more programming languages and you can also mostly configure text editors with extensions to become IDEs.
+
+###  Terminal based editors
+
+#### Nano
+Nano is a very simple terminal based editor that is easy to use, shows it's help at the bottom, is installed on basically every unix system but for example doesn't support many features for programming like syntax highlighting.
+
+#### (neo)vim
+(Neo)Vim is the text editor you saw me using in the lecture, it is hard to use for beginners, because it works very differently to other editors you might be used to. It is mode based and is very powerfull especially with lots of extensions.
+
+#### Emacs
+The other large, very extensible terminal text editor. Some people joke it has so many features that it is its own operating system, only lacking a good text editor.
+
+#### Graphical Text Editors
+There are lots, but currently I would just recommend you try out Microsoft's Visual Studio Code (https://code.visualstudio.com/), or the open source builds of it without telemetry called VS Codium (https://code.visualstudio.com/)
```

Comment:
(typo) the second link should be: https://vscodium.com/

Reply:
will update, thanks